### ISSUE TITLE
Use colors + icons for the FF containers

### ISSUE THREAD

**Issue by michaelfecher** [Systanics/CloudGlance#11]
Currently, it's hard to grasp the "yellow" color + "fingerprint" icon on the light gray tab in Firefox, at least it's yellow for me in the default theme of FF. 

![image](https://user-images.githubusercontent.com/13333848/192993050-9f5a0af1-043d-4c2c-bc01-457227e581e7.png)

If you have multiple tabs with accounts open, it's hard to spot it directly which account is in each tab.
To enable a better readability and understandability, it would be nice to select the color + icon in the account profile setting.

The containers itself support custom colors and some other icons, as it can be seen here:
https://support.mozilla.org/en-US/kb/containers
and in the settings:
about:preferences#containers

**Comment by rehanvdm** [comment]
Yeah, I tried to set this via the CLI Extension some time back and it didn't work. Let me look into it again. In the interim, you can always change the settings of the container after it has been created. Cloud Glance will not override it.
![multi_container_1](https://user-images.githubusercontent.com/22810856/193440035-a8644281-24b8-4d0f-8061-dc8a97c14e19.png)
![multi_container_2](https://user-images.githubusercontent.com/22810856/193440041-994a2ab7-6db8-4350-898a-96bc479d0ff4.png)

**Comment by rehanvdm** [comment]
I have opened an issue on the extension used to open URLs within FF containers here https://github.com/honsiorovskyi/open-url-in-container/issues/115 I do not think there will be much movement on it, so not holding my breath. At some point, I was thinking of creating a browser extension specifically for Cloud Glance that does the same as that extension. But that is not a priority at the moment. 

In the meantime, I made it open in with random color for the first time, if the container does not exist yet. The current browser plugin can only set the color and icon if it does not exist yet, so I don't want to include this as an option within Cloud Glance as it will have no effect after the first time the container is created. So the color and icon need to be changed within the browser as shown above for now.
